make/test/JtregNativeHotspot.gmk

```diff
@@ -857,6 +857,11 @@ else
       exeinvoke.c exestack-gap.c exestack-tls.c libAsyncGetCallTraceTest.cpp
 endif
 
+ifeq ($(call And, $(call isTargetOs, linux) $(call isTargetCpu, aarch64)), false)
+  BUILD_HOTSPOT_JTREG_EXCLUDE += libTestSVEWithJNI.c
+endif
```

Comment:
I would have put this inside the existing Linux section  ie move to before the endif at line 858, and then just check for aarch64.

Reply:
It's the other way around - this test is excluded for all platforms _but_ linux-aarch64 (note the `false`).

Reply:
Ah - sorry.